packages/calcite-components/src/components/popover/popover.tsx

```diff
@@ -589,7 +590,7 @@ export class Popover
           <div
             class={{
               [CSS.hasHeader]: !!heading,
-              [CSS.container]: true,
+              [CSS.headerContainer]: true,
```

Comment:
This class was modified, but previous `.container` selectors have not been updated. Is this intentional?

Reply:
No, apologies. I modified the test selectors, but haven't yet worked on the css.